### ISSUE TITLE
Datepicker components does not display

### ISSUE THREAD

**Issue by dream-corner** [Attendize/Attendize#681]
Hello,
Locally on my pc the date system works for event creation. By cons I just installed on a server vps ovh and the date system is no longer displayed. I do not understand, thanks for your help.

[http://zupimages.net/viewer.php?id=19/36/u2uu.png](url)

**Comment by publicarray** [comment]
If you look at the browser console is there any error? What format did you use? Did you change any variables between the VPS and your PC?

Please copy the following variables from your `.env` file
```
DEFAULT_DATEPICKER_SEPERATOR=
DEFAULT_DATEPICKER_FORMAT=
DEFAULT_DATETIME_FORMAT=
```

**Comment by yan-embtel** [comment]
![image](https://user-images.githubusercontent.com/55192106/64696381-e22bb980-d4bb-11e9-9789-129568f2b1be.png)

Date Pickers are not working anymore in CHROME, please visit http://attendize.website/organiser/9226/events?first_run=1

**Comment by publicarray** [comment]
Hi @yan-embtel The Online demo is running an out of date version, this was fixed in #612
